src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -16,13 +21,19 @@
   set_fact:
     extended_packageset: "{{ extended_packageset | default('no') }}"
 
+- name: 'Install packages via rpm-ostree'
+  include_tasks: 'rpm-ostree.yml'
+  when: rpm_ostree.stat.exists
```

Comment:
Maybe change the logic in https://github.com/SSSD/sssd-ci-containers/blob/master/src/ansible/roles/packages/tasks/main.yml#L1 instead of adding all the conditions here?

* if rpm-ostree, include $distro.rpm-ostree.yml